src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue.IntegrationTests/DbSetupScripts/6_PeriodEnd.Deds.PaymentsDue.Add_APIIndexes.sql

```diff
@@ -0,0 +1,20 @@
+IF NOT EXISTS(SELECT NULL FROM sys.indexes WHERE name='IX_RequiredPayments_AccountId')
+BEGIN
+	CREATE NONCLUSTERED INDEX [IX_RequiredPayments_AccountId]
+		ON [PaymentsDue].[RequiredPayments] ([AccountId])
+END
+GO
+
+IF NOT EXISTS(SELECT NULL FROM sys.indexes WHERE name='IX_Payments_CollectionPeriod')
+BEGIN
+	CREATE NONCLUSTERED INDEX [IX_Payments_CollectionPeriod]
+		ON [Payments].[Payments] ([CollectionPeriodYear], [CollectionPeriodMonth])
+END
+GO
+
+IF NOT EXISTS(SELECT NULL FROM sys.indexes WHERE name='IX_Earnings_RequiredPaymentId')
+BEGIN
+	CREATE NONCLUSTERED INDEX [IX_Earnings_RequiredPaymentId]
+		ON [PaymentsDue].[Earnings] ([RequiredPaymentId])
+END
+GO
```

Comment:
When running the provider events query, the query plan suggested the following:

missing statistics hints on Payments.Payments.RequiredPaymentId
missing statistics hints on Payments.Payments.CollectionPeriodMonth

Also, it had issue with the type conversion of the payments.paymentid from a uniqueidentifier to a string with the following message:

Type conversion in expression try_cast may affect cardinality estimate in query plan choice

The sql script itself isn't part of this pull request so that can be ignored for now (but should probably be looked at in the future. Similarly, if there isn't any massive performance hit, the stats hints can be put aside for another time.